firmware/esp32/gdoor/README.md

```diff
@@ -19,19 +19,40 @@ as a baseline.
 Currently only the serial port is implemented.
 Connect to the port with baudrate 115200.
 
+### Serial Commands
+The following serial commands can be send to the bus adapter:
+- `*debug`: Activate debug mode with more verbose output
+- `*normal`: Switch back to normal mode, with limited output
+
+### Receive bus events/commands
 On each bus communication you will receive a JSON formatted string with the following fields:
-- data: The bus data received. Array of bytes.
+- action: The action of the bus message like e.g. `OPEN_DOOR`. For details please look into `gdoor_data.cpp`.
+- type: The hardware type of the message sender. For details please look into `gdoor_data.cpp`.
+- source: Bus address of message sender, 3 Bytes.
+- destination: Bus address of message destination, 3 Bytes. This field is not contained in each bus message in such cases it is set to 0.
+- parameters: Parameter values depend on action and bus structure. E.g. it can be the ID of which button was pressed, 2 Bytes.
+- busdata: Bus data as hex value. You can send this value back to the serial port to duplicate a bus message.
+- event_id: Counter value. It counts up for each new bus message.
+
+If you switch on debug mode, you will also receive:
 - raw: The raw counting values of each high/low bit. Array of bytes where each byte represents the number of pulses of a individual bit.
 - hexvalue: This is the same value as data, but formatted, so that you can copy and paste it to the serial terminal,
```

Comment:
I do like the new wording instead of `hexvalue`. But shouldn't we call it `busdata` in both cases, meaning normal and debug mode?